### PR TITLE
Document the ProductOpener::URL.pm module

### PR DESCRIPTION
**Pull request by areeshatariq**
**Description:**

Documentation of the ProductOpener::URL.pm module in perlpod format

**Related issues and discussion:** #3044

### CODE REVIEW

File: lib/ProductOpener/URL.pm [hunk 1]
```diff
@@ -42,6 +66,22 @@ use experimental 'smartmatch';
...
 
+=head1 FUNCTIONS
+
+=head2 format_subdomain( SUBDOMAIN )
+
+C<format_subdomain()> returns URL on the basis of subdomain and scheme (http/https)
+
+=head3 Arguments
+
+A scalar variable needs to be passed as an argument.
```
Comment (stephanegigandet) [inline lib/ProductOpener/URL.pm]: In addition to its type, we need to indicate was the argument is.

```suggestion
A scalar variable to indicate the subdomain (e.g. "us") needs to be passed as an argument. 
```

Comment (stephanegigandet) [reply lib/ProductOpener/URL.pm]: Same for the other functions.

**Comment by areeshatariq** [comment]
@stephanegigandet I've made the suggested changes. Does it look fine now?

File: lib/ProductOpener/URL.pm (hunk 1 above)
Comment (areeshatariq) [reply lib/ProductOpener/URL.pm]: Done

**Pull request merged**